backend/workflow_manager/endpoint/queue_utils.py

```diff
@@ -0,0 +1,19 @@
+import logging
+from typing import Any
+
+from utils.constants import Common
+
+from unstract.connectors.queues import connectors as queue_connectors
+from unstract.connectors.queues.redis import RedisQueue
+
+logger = logging.getLogger(__name__)
+
+
+class QueueUtils:
+    @staticmethod
+    def get_queue_class(
+        connector_id: str, connector_settings: dict[str, Any]
+    ) -> RedisQueue:
+        connector = queue_connectors[connector_id][Common.METADATA][Common.CONNECTOR]
+        connector_class: RedisQueue = connector(connector_settings)
+        return connector_class
```

Comment:
NIT: @vishnuszipstack technically we are obtaining an instance of the `RedisQueue` class here - so I suggest renaming this function and its variables as `get_queue_inst` and `redis_queue_inst` or anything along this lines

Reply:
Done @chandrasekharan-zipstack